### DISCUSSION TITLE
Start Minio with a bucket with previous objects

### DISCUSSION THREAD

**Discussion by Stenfrank** [minio/minio#16057] [Q&A] [answered]
I am starting with Minio through docker I already have a folder with data but when running minio it only detects the folders and not the files that are inside the folders.

I clarify the folders and files were not created by the API already existed, is this possible?

**Comment by harshavardhana** [comment, marked as answer]
You cannot do that anymore @Stenfrank - we have deprecated this support.

**Reply by Stenfrank** [reply]
Is there a way to force the Bucket for the first time or what version of docker can I use for this functionality?
